src/terraform/index_test.go

```diff
@@ -69,7 +69,7 @@ provider "aws" {
 }
 
 module "aws" {
-  source = "./aws"
+  source = "git@github.com:Originate/exosphere.git//src//terraform//modules//aws?ref=8786f912"
```

Comment:
I think for now when they're all the same that makes sense. But in the future it's possible modules would change individually of each other, in which case having a single variable might be problematic. Or we could be sourcing modules from different directories.

I also like that the commit hash is hardcoded into the module themselves, so there isn't margin for error when generating or interpolating the git url.